jobrunner/Dockerfile

```diff
@@ -8,5 +8,4 @@ RUN git clone --depth=1 https://github.com/wikimedia/mediawiki-services-jobrunne
 WORKDIR /jobrunner
 RUN composer install --no-dev
 COPY ./config.json /jobrunner/config.json
-ENTRYPOINT php /jobrunner/redisJobChronService --config-file=/jobrunner/config.json
```

Comment:
I think we need this;-)

Reply:
Since jobrunner and rescheduler use different entrypoint scripts I was planning to define it here, I haven't tested it yet though: https://github.com/MaRDI4NFDI/portal-k8s/blob/6aa51ddc3cd9d281c2da33e056e11cbc3b90214e/charts/jobrunner/values.yaml#L25